packages/trace-viewer/src/ui/modelUtil.ts

```diff
@@ -141,7 +139,7 @@ export class MultiTraceModel {
     return this.errors.filter(e => !!e.message).map((error, i) => ({
       stack: error.stack,
       message: error.message,
-      context: this.attachments.find(a => a.name === `_error-context-${i}`),
+      context: this.attachments.find(a => a.name === `error-context-${i}`),
```

Comment:
this will fail for the old traces that used `_error-context*` name, do we need a migration?